api.json

```diff
@@ -0,0 +1,761 @@
+            "tuesday?": "Boolean",
+            "wednesday?": "Boolean",
+            "thursday?": "Boolean",
+            "friday?": "Boolean",
+            "saturday?": "Boolean",
+            "sunday?": "Boolean",
+            "deleted": "Boolean"
+          }
+        },
+        "PUT": {
+          "description": "Update visit activity for an organistion",
+          "auth": true,
+          "scope": [],
+          "query": { 
+            "name": "String",
+            "category": "String",
+            "monday": "Boolean",
+            "tuesday": "Boolean",
+            "wednesday": "Boolean",
+            "thursday": "Boolean",
+            "friday": "Boolean",
+            "saturday": "Boolean",
+            "sunday": "Boolean"
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "category": "String",
+            "monday?": "Boolean",
+            "tuesday?": "Boolean",
+            "wednesday?": "Boolean",
+            "thursday?": "Boolean",
+            "friday?": "Boolean",
+            "saturday?": "Boolean",
+            "sunday?": "Boolean"
+          }
+        },
+        "DELETE": {
+          "description": "Soft delete visit activity for an organistion",
+          "auth": true,
+          "scope": [],
+          "query": { 
+            "name": "String",
+            "category": "String",
+            "monday?": "Boolean",
+            "tuesday?": "Boolean",
+            "wednesday?": "Boolean",
+            "thursday?": "Boolean",
+            "friday?": "Boolean",
+            "saturday?": "Boolean",
+            "sunday?": "Boolean"
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "category": "String",
+            "monday?": "Boolean",
+            "tuesday?": "Boolean",
+            "wednesday?": "Boolean",
+            "thursday?": "Boolean",
+            "friday?": "Boolean",
+            "saturday?": "Boolean",
+            "sunday?": "Boolean"
+          }
+        }
+      },
+      "/:id/feedback": {
+        "GET": {
+          "description": "Retreive feedback for one organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "since": "String",
+            "until": "String"
+          },
+          "response": {
+            "-1": "Number",
+            "0": "Number",
+            "1": "Number",
+            "since": "String",
+            "until": "String"
+          }
+        },
+        "POST": {
+          "description": "Send feedback for one organisation",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "feedbackScore": "Number"
+          },
+          "response": {
+            "feedbackScore": "Number",
+            "organisationId": "Number"
+
+          }
+        }
+      },
+      "/:id/users": {
+        "GET": {
+          "description": "Retreive all users in an organisation",
```

Comment:
Should probably also have a filter by user type? Or instead have aliases `/:id/volunteers` and `/:id/visitors` and `/:id/admins`?